lib/errors.js

```diff
@@ -23,7 +23,10 @@ class CliError extends Error {
         Error.captureStackTrace(this, this.constructor);
 
         this.context = options.context || {};
+        // TODO: Do something with the original error message which we can receive now as the `err` property
+        this.err = options.err || {};
```

Comment:
@kirrg001 What is your opinion on going further here? Does it make more sense to pick the possible error properties where they occur? For example, when we know the error [here](https://github.com/TryGhost/Ghost-CLI/pull/595/files#diff-f158eddc93a4bd540b26f3dc0da0527dR55) should almost always be a request error, shall we rely on that and pick the properties that `got` returns? Or just pass the full original error and log it into the log files?

Reply:
I would try to keep this as simple as possible.

e.g. log custom CLI error, log original error if exists (e.g. iterate over the properties and log each field). Maybe we can only log the original error into the debug file.

custom.message
custom.stack
original.message
original.stack